autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/SectionSelect/SectionSelect.tsx

```diff
@@ -56,8 +73,7 @@ const SectionSelect: React.FC<SectionSelectProps> = ({ id }): JSX.Element => {
         {formatMeetingDays(mtg)}
       </span>
       <span className={styles.meetingDetailsText}>
-        {`${formatTime(mtg.startTimeHours, mtg.startTimeMinutes)}
-        - ${formatTime(mtg.endTimeHours, mtg.endTimeMinutes)}`}
+        {getMeetingTimeText(mtg)}
```

Comment:
I think you could just have `getMeetingTimeText` return the text instead of a span with the text, to reduce unnecessary clutter in the DOM. The way it is right now, there's a `<span>` inside of a `<span>`

Reply:
Yeah that makes a lot more sense, squashed into the first commit